### DISCUSSION TITLE
Position popper on top of the reference element

### DISCUSSION THREAD

**Discussion by klausXR** [floating-ui/floating-ui#2093] [Q&A]
Is there a way to have my popper element positioned on top of the reference element, for example starting from its top right corner?

Examples, not relevant to popper, but you can see an example how the dropdown of the select appears on top of the input.

https://vuetifyjs.com/en/components/selects/

**Comment by atomiks** [comment]
You can use `offset` where the distance offset is equal to the reference element's `height * -1`
